src/libdrakvuf/vmi.c

```diff
@@ -612,6 +614,8 @@ event_response_t cr3_cb(vmi_instance_t vmi, vmi_event_t* event)
     trap_info.proc_data.pid       = proc_data.pid;
     trap_info.proc_data.ppid      = proc_data.ppid;
     trap_info.proc_data.userid    = proc_data.userid;
+    if (drakvuf_get_os_type(drakvuf) == VMI_OS_LINUX)
+        trap_info.proc_data.tid    = proc_data.tid;
```

Comment:
You can remove this

Reply:
There should be no OS checks in this level, need to remove `if (drakvuf_get_os_type(drakvuf) == VMI_OS_LINUX)`